react-native/react/actions/tracker.js

```diff
@@ -18,6 +18,55 @@ type TrackerActionCreator = (dispatch: Dispatch, getState: () => {tracker: RootT
 
 // TODO make actions for all the call back stuff.
 
+export function startTimer (): (dispatch: Dispatch, getState: any) => void {
+  return (dispatch, getState) => {
+    // Increments timerActive as a count of open tracker popups.
+    dispatch({type: Constants.startTimer})
+    const timerActive = getState().tracker.timerActive
+    if (timerActive === 1) {
```

Comment:
I think this is fine for now but i could imagine this not working if the dispatch / reducer flow isn't sync (like if we used a batching middleware).